modules/codegen-plugin/src/sbt-test/codegen-plugin/aws-newtype-flatten/build.sbt

```diff
@@ -0,0 +1,17 @@
+lazy val root = (project in file("."))
+  .enablePlugins(Smithy4sCodegenPlugin)
+  .settings(
+    scalaVersion := "2.13.10",
+    Compile / smithy4sAllowedNamespaces := List(
+      "aws.api",
+      "aws.auth",
+      "aws.customizations",
+      "aws.protocols",
+      "com.amazonaws.dynamodb",
+    ),
+    libraryDependencies ++= Seq(
+      "com.disneystreaming.smithy4s" %% "smithy4s-core" % smithy4sVersion.value,
+      "com.disneystreaming.smithy"   % "aws-dynamodb-spec" % "2023.02.10"
```

Comment:
see my comment [here](https://github.com/disneystreaming/smithy4s/pull/1110/files#r1270308761). Compiling the full dynamodb generated code is detrimental to build times.